src/modules/gcal.js

```diff
@@ -0,0 +1,28 @@
+import Axios from "axios";
+import Moment from "moment";
+
+const CALENDAR_ID = process.env.REACT_APP_GOOGLE_CALENDAR_ID;
+const API_KEY = process.env.REACT_APP_GOOGLE_CALENDAR_API_KEY;
```

Comment:
These will need to be populated on the deployment machine, correct? I'll make a note on that issue #31 to make sure to add them

Reply:
Yes! Thanks for doing so :).